projects/account/src/app/core/http/profile.service.ts

```diff
@@ -87,4 +89,18 @@ export class ProfileService {
         const codedPassword = btoa(newPassword);
         return this.http.put(CHANGE_PASSWORD_URL, {password: codedPassword});
     }
+
+    validateEmailAddress(token: string): Observable<any> {
+        const queryParams = {platform: 'Internal', token: token};
+        return this.http.get(VALIDATE_EMAIL_URL, {params: queryParams});
+    }
+
+    changeEmailAddress(newEmailAddress: string) {
+        const codedEmailAddress = btoa(newEmailAddress);
+        return this.http.put(CHANGE_EMAIL_ADDRESS_URL, {token: codedEmailAddress});
+    }
+
+    verifyEmailAddress(newEmailAddress: string) {
+        return this.http.put<string>(VERIFY_EMAIL_URL, {token: newEmailAddress});
```

Comment:
looks like a typo with `<string>`

Reply:
not a typo, it is how you indicate that an observable string object will be returned